actions/llm-action-error-reporter/aer.sh

```diff
@@ -4,24 +4,31 @@
 input=$(cat)
```

Comment:
I'd set the shebang on all of these scripts like this along with setting it to fail on errors, unassigned vars, and when a piped command fails like:

```
#!/usr/bin/env bash

set -euo pipefail
```